basic/src/test/java/org/jboss/hal/testsuite/test/configuration/container/EJB3BeanPoolTestCase.java

```diff
@@ -93,7 +93,7 @@ public void createBeanPool() throws Exception {
                 .timeoutUnit("MINUTES")
                 .finish();
         Assert.assertTrue("Window should be closed", result);
-        new ResourceVerifier(BEAN_POOL_TBA_ADDRESS, client).verifyExists();
+        new ResourceVerifier(BEAN_POOL_TBA_ADDRESS, client).verifyExists("Fails because of https://issues.jboss.org/browse/HAL-1196");
```

Comment:
Please change "Fails because of ..." to something like "Probably fails because of ..."